### PR TITLE
feat(schemas): add domains table

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add `domains` table, see [tech design doc](https://www.notion.so/silverhand/Custom-Domain-5d13ab5693304745bf4ac9a47e5dd797) for more info.

## Testing

Local tested with the alter script.

## Checklist

- [x] `.changeset` LOG-6185
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/schemas/alterations/next-1684739802-create-hook-id-index-for-logs copy.ts
Comment (gao-sun) [inline packages/schemas/alterations/next-1684739802-create-hook-id-index-for-logs copy.ts:1]: mistake?

Comment (wangsijie) [reply packages/schemas/alterations/next-1684739802-create-hook-id-index-for-logs copy.ts:1]: updated

File: packages/schemas/tables/domains.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+create table domains (
+  tenant_id varchar(21) not null
+    references tenants (id) on update cascade on delete cascade,
+  id varchar(21) not null,
+  domain varchar(256) not null,
```
Comment (simeng-li) [inline packages/schemas/tables/domains.sql:5]: unique?

Comment (wangsijie) [reply packages/schemas/tables/domains.sql:5]: updated

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**